.github/workflows/ci.yml

```diff
@@ -68,6 +68,14 @@ jobs:
         run: |
           ./meta-dts/scripts/generate-ipxe-menu.sh ${{steps.dts-ver.outputs.DTS_VER}}
           scp -i ~/.ssh/dts-ci-key dts.ipxe builder@10.1.40.2:boot/dts/
+      - name: Trigger signing
+        shell: bash
+        run: |
+          DTS_VER="${{steps.dts-ver.outputs.DTS_VER}}"
+          git clone ssh://git@git.3mdeb.com:2222/3mdeb/dts-release-cicd-pipeline.git
+          cd dts-release-cicd-pipeline
+          git tag `echo $DTS_VER | tr -d "v"`
+          git push origin main --tags
```

Comment:
Here operations on private repo is performed. I assume that the server where the runner runs this pipeline has permissions to manipulate on our private repos.

Reply:
@DaniilKl so we need an additional SSH key in this pipeline which will allow this cloning. Did you test this? While testing we do not need pushing any tags, just cloning will be enough.

Reply:
@TomaszAIR I did not tested it ([comment](https://github.com/Dasharo/meta-dts/pull/41#issuecomment-1736998331)). 

Yep, we need some key to get access to our repos.

Reply:
@DaniilKl you should have access to both (gitea and builder), please add SSH key via secret, test it and I will later replace that key with a valid one

Reply:
@DaniilKl jop runs on any tag pushed so you could e.g. bump version to 1.2.10-rc1, then create such tag and push it. Or, even better, just for tests comment out any commands that build/clone artifacts to the FTP server.